feat(content): Patir 2 (#9959)
endless-sky/endless-sky: data/map beyond patir.txt

Reviewer: The devs are talking about splitting the systems file into regions because of how big it's gettering (again), so I'm chill with leaving this in its own file.

Author: Added government uninhabited

		distance 4316.1251
		period 10.1251
		offset 180

system Mitera
	pos 9957.5 -92.5
	haze _menu/haze-mitera
	habitable 3645
	belt 1464
	fleet "Magic Asteroid" 900
	object
		sprite "star/black hole 4 core"
		period 1
	object
		sprite "star/black hole 4"
		period 1
	object "One-Way Wormhole Entrance"
		sprite "planet/wormhole-blue"
		distance 4580.2286
		period 11.3382
		offset 180

system Sarifa
	pos 10136.5 -406.5
	haze _menu/haze-sarifa
	habitable 5486.68
	belt 1757
	fleet "Magic Asteroid" 900
	object
		sprite "star/f8"
		distance 549
		period 1
		offset 78
	object
		sprite "star/black hole star core"
		period 1
	object
		sprite "star/black hole star"
		period 1

system Thepa
	pos 9975.5 -2.5
	haze _menu/haze-maithi
	habitable 320
	belt 1793
	object
		sprite "star/black hole 6 core"
		period 1
	object
		sprite "star/black hole 6"
		period 1

system Balnii
	pos 10268.3 -359.3
	haze _menu/haze-sarifa
	arrival 1100
	habitable 360
	belt 1497
	fleet "Magic Asteroid" 900
	hazard "Balnii Black Hole Event Horizon" 1
	hazard "Black Hole Gravity" 2
	object
		sprite "star/big black hole core"
		period 1
	object
		sprite "star/big black hole"
		period 1

planet "One-Way Wormhole Entrance"

> 	description ``

Reviewer 2: What is this description for? NPCs land if there is a `spaceport`, but adding a description won't do anything.

	wormhole "One-Way Wormhole Entrance"

planet "One-Way Wormhole Exit"
	description ``
	wormhole "One-Way Wormhole Exit"

wormhole "One-Way Wormhole Entrance"
	mappable
	link Mitera Athiri
	color "wormholes: beyond Patir"

wormhole "One-Way Wormhole Exit"
	mappable
	link Maithi Ghila
	color "wormholes: beyond Patir"

planet "Chanai Structure"

> 	government "Magic Asteroid"

Reviewer 2: The government is usually at the bottom of the planet definition.
```suggestion
```

	landscape "land/station49"

> 	description `The alien station is as vast as it is mysterious, concealing more secrets than you could ever hope to find. Its only dishernible feature is a giant barrel, but what it was built for - or what it could still do - is impossible to say.`

Reviewer 3: ```suggestion
	description `The alien station is as vast as it is mysterious, concealing more secrets than you could ever hope to find. Its only discernible feature is a giant barrel, but what it was built for - or what it could still do - is impossible to say.`
```

> 	description `The alien station is as vast as it is mysterious, concealing more secrets than you could ever hope to find. Its only discernible feature is a giant barrel, but what it was built for - or what it could still do - is impossible to say.`

Reviewer 2: ```suggestion
	description `This alien station is as vast as it is mysterious, concealing more secrets than you could ever hope to find. Its only discernible feature is a giant barrel, but what it was built for - or what it could still do - is impossible to say.`
	government "Magic Asteroid"
```

		period 1
	object
		sprite "star/black hole 4"
		period 1
	object "One-Way Wormhole Entrance"
		sprite "planet/wormhole-blue"
		distance 4580.2286
		period 11.3382
		offset 180

system Sarifa
	pos 10136.5 -406.5
	haze _menu/haze-sarifa
	habitable 5486.68
	belt 1757
	fleet "Magic Asteroid" 900
	hazard "Beyond Patir Shield Eraser" 1
	hazard "Beyond Patir Black Hole Event Horizon" 1
	hazard "Black Hole Gravity" 1
	object
		sprite "star/f8"
		distance 549
		period 1
		offset 78
	object
		sprite "star/black hole star core"
		period 1
	object
		sprite "star/black hole star"
		period 1

system Thepa
	pos 9975.5 -2.5
	haze _menu/haze-maithi
	habitable 320
	belt 1793
	hazard "Beyond Patir Shield Eraser" 1
	hazard "Beyond Patir Black Hole Event Horizon" 1
	hazard "Black Hole Gravity" 1
	object
		sprite "star/black hole 6 core"
		period 1
	object
		sprite "star/black hole 6"
		period 1

planet "One-Way Wormhole Entrance"
	wormhole "One-Way Wormhole Entrance"

planet "One-Way Wormhole Exit"
	wormhole "One-Way Wormhole Exit"

wormhole "One-Way Wormhole Entrance"
	mappable
	link Mitera Athiri
	color "wormholes: beyond Patir"

wormhole "One-Way Wormhole Exit"
	mappable
	link Maithi Ghila

> 	color "wormholes: beyond Patir"

Author: One is the only way to enter the northern sector of that galaxy, while the other is the only way to exit it. I'm not sure how to clarify that better

> 	color "wormholes: beyond Patir"

Reviewer 4: Maybe "Beyond Patir North Entrance" and "Beyond Patir North Exit", then?

system Thepa
	pos 9975.5 -2.5
	haze _menu/haze-maithi
	habitable 320
	belt 1793
	object
		sprite "star/black hole 6 core"
		period 1
	object
		sprite "star/black hole 6"
		period 1

> system Balnii

Reviewer 5: :tada:
